src/loc.ml

```diff
@@ -70,10 +70,28 @@ let file_line path n =
       input_line ic
     )
 
+let file_lines path ~start ~stop =
+  Io.with_file_in ~binary:true path
+    ~f:(fun ic ->
+      let rec aux acc lnum =
+        if lnum > stop then
+          List.rev acc
+        else if lnum < start then
+          let _ = input_line ic in
```

Comment:
more common is to use `ignore` here btw.

Reply:
i've switched to using `ignore` now